src/base_types.ts

```diff
@@ -12,6 +12,7 @@ export interface ContextParams {
   SESSION_STORAGE?: SessionStorage;
   LOG_FILE?: string;
   USER_AGENT_PREFIX?: string;
+  PRIVATE_APP_STOREFRONT_ACCESS_TOKEN?: string;
```

Comment:
Slightly confused by this naming. Do only private apps use the SF API?

Reply:
There are 2 ways to get access tokens: 
1. Via your admin, where you grant a private app SF permission and get a hard-coded token (which would go into this field)
2. The example in the PR description, where you can create public tokens for a sales channel

I wanted to make it very clear that this field was meant only for private apps, which is why the name is quite the mouthful.

Reply:
How do you obtain those public tokens for a sales channel? and how would they be used in an app?  Are they retrieved via an API or does the app developer need to load them from the environment/database/files/etc.?

Reply:
Public apps will use the second path above, by querying the `storefront_access_token` endpoint the Admin API. The code I used in the example is for public apps.